### DISCUSSION TITLE
Google Analytics for Listed Blog

### DISCUSSION THREAD

**Discussion by Nizumzen** [standardnotes/forum#3597] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

Yes

**Is your feature request related to a problem? Please describe.**

Yes. It would help people to see if their Listed blog is getting visitors or not.

**Describe the solution you'd like**

Add a setting asking for your Google Analytics website code and then embed the Google Analytics code into the note when it is published to the blog.

**Describe alternatives you've considered**

Having some method that allows you to inject JavaScript into a Listed post would allow people to do their own thing. For instance they might want to use another platform than Google Analytics.

**Comment by dechristopher** [comment]
Custom JS snippets could also allow for self-hosted and other analytics platforms to be used, like Plausible for example.
